readme.md

```diff
@@ -106,7 +112,18 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get('/get', function(req, res) {
+  res.send("GET");
+});
+app.post('/post', function(req, res) {
+  res.send("POST");
+});
+app.delete('/delete', function(req, res) {
+  res.send("delete");
+});
+app.put('/put', function(req, res) {
+  res.send("put");
+});
```

Comment:
This is very close correct, only change I'd make is to send the name of the restful routes, not the HTTP verbs:
* Get -> Show (or index)
* Post -> Create
* Delete -> Delete/Destroy
* Put -> Update